docs/maatregelen/0_org_gov_06_governance_inrichten_three_lines.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Richt een algoritmegovernance in met three lines of defence
+toelichting: Richt een algoritmegovernance in met three lines of defence
+
+levenscyclus:
+- organisatieverantwoordelijkheden
+
+onderwerp:
+- governance
+
+rollen:
+- projectleider
+# - beleid-en-advies
+
+---
+
+## Maatregel
```

Comment:
```suggestion
## Maatregel
Richt een algoritmegovernance in met three lines of defence
```

Reply:
Of gaat dit eigenlijk voer het inrichten van intern toezicht? En is de maatregel dan: 

Richt processen in voor intern toezicht op algoritmes.

Three lines of defence is dan de best practice? 4 lines of defence ben ik ook vaker tegen gekomen. 4e lijn is dan extern toezicht of een onafhankelijke audit